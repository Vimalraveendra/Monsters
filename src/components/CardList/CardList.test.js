import { shallow } from "enzyme";
import React from "react";
import CardList from "./CardList";
import expectExport from "expect";

it("should render the CardList Component", () => {
  const mockMonsters = [
    {
      id: 1234,
      name: "John",
      email: "John@gmail.com"
    }
  ];
  const wrapper = shallow(<CardList monsters={mockMonsters} />);
  expectExport(wrapper).toMatchSnapshot();
});
