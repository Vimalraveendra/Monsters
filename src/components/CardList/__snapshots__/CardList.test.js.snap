// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the CardList Component 1`] = `ShallowWrapper {}`;
